react-basic-practice/src/components/Button/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { Button as ChakraButton, ButtonProps } from "@chakra-ui/react";
+import { borderRadius } from "../../themes/metrics";
+
+interface CustomButtonProps extends ButtonProps {
+  variant: "primary" | "secondary" | "default";
```

Comment:
consider use optional type. And remove `default` type. We don't need it.

Reply:
i done